File: tests/fixtures/valid_input.py
def parse_port(value: str) -> int:
    """@generate
    behavior:
      strip(value)
      require value matches digits
      require 1 <= int(value) <= 65535
      return int(value)
    examples:
      parse_port("80") == 80
      parse_port("443") == 443
      parse_port("0") raises ValueError
    constraints:
      no_imports
      no_network
      no_filesystem
    """
    ...

File: tests/fixtures/ts/valid_input.ts
/**
 * @generate
 * behavior:
 *   strip(value)
 *   require value matches digits
 *   require 1 <= parseInt(value) <= 65535
 *   return parseInt(value)
 *
 * examples:
 *   parsePort("80") == 80
 *   parsePort("443") == 443
 *   parsePort("0") raises ValueError
 *
 * constraints:
 *   no_imports
 *   no_network
 *   no_filesystem
 */
export function parsePort(value: string): number {
  return 0;
}
